packages/core-ethereum/crates/core-ethereum-indexer/src/handlers.rs

```diff
@@ -334,6 +334,7 @@ where
                     return Err(CoreEthereumIndexerError::ChannelDoesNotExist);
                 }
             }
+            _ => todo!()
```

Comment:
It'd be safer to enumerate all enum cases and then or them into the todo. This way more than is desired could be hidden.

Reply:
Thanks. Learnt a new thing! I got the time to implement handlers. Please review again :D